db/schema.rb

```diff
@@ -845,9 +845,33 @@
     t.index ["access_grant_id"], name: "index_oauth_openid_requests_on_access_grant_id"
   end
 
+  create_table "payment_intents", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
+    t.string "holder_type"
+    t.bigint "holder_id"
+    t.string "payment_record_type"
+    t.bigint "payment_record_id"
+    t.text "client_secret"
+    t.text "error_details"
+    t.datetime "created_at", precision: nil, null: false
+    t.datetime "updated_at", precision: nil, null: false
+    t.integer "initiated_by_id"
+    t.datetime "confirmed_at", precision: nil
+    t.string "confirmation_source_type"
+    t.bigint "confirmation_source_id"
+    t.datetime "canceled_at", precision: nil
+    t.string "cancellation_source_type"
+    t.bigint "cancellation_source_id"
+    t.string "wca_status"
+    t.index ["cancellation_source_type", "cancellation_source_id"], name: "index_stripe_payment_intents_on_canceled_by"
+    t.index ["confirmation_source_type", "confirmation_source_id"], name: "index_stripe_payment_intents_on_confirmed_by"
+    t.index ["holder_type", "holder_id"], name: "index_stripe_payment_intents_on_holder"
+    t.index ["initiated_by_id"], name: "fk_rails_2dbc373c0c"
```

Comment:
Yikes, not a nice index name. How is this coming into being?

Reply:
(I see it was there in the old diff as well. Feel free to ignore it if you cannot come up with a good and easy solution in 15 minutes)

Reply:
Research suggests that Rails automatically creates these indexes when polymorphic associations are specified, but doesn't rename them automatically if a table name is changed (fair enough) - so I've added a `rename_index` line in the migration file